### ISSUE TITLE
Enable hardware acceleration on Android 8/8.1 devices

### ISSUE THREAD

**Issue by tuomas2** [AndBible/and-bible#1765]
As a workaround to #1756, WebView hardware acceleration was disabled. 
This causes some degradation of performance on those devices. Try to find a better solution for those devices.

**Comment by tuomas2** [comment]
Was googling a little for a better solution, but did not find any. Lowering priority now.

**Comment by timbze** [comment]
Has a bug been filed with Android? I guess it's possible that it could take quite some time to get resolved.

**Comment by timbze** [comment]
Another user on 8.1 who is discouraged on how slow the app is now. 

https://support.andbible.org/scp/tickets.php?id=873

**Comment by tuomas2** [comment]
@agrogers please note!^^

**Comment by agrogers** [comment]
Noted it in the ticket for Wayne.

___________________________
*www.TenOutOfTen.org* <https://www.TenOutOfTen.org>
***@***.***
(+61) 0477 174 193 (Australia)
(+95) 09 250018669 (Myanmar)

**Comment by tuomas2** [comment]
This ticket issue affects scrolling speed etc, but probably not overall app speed. 

Overall app speed has become slower because of features and the chosen technology to implement them. That's simply the tradeoff. There can be something to improve, but I don't think we can do miracles here. I feel it is "fast enough" even on the slowest devices I have (my tablet is pretty slow and my wife has even slower and older tablet that I use for testing).

**Comment by tuomas2** [comment]
Should check if more recent chrome fixes issue on 8 and 8.1

**Comment by tuomas2** [comment]
Tested most recent chrome without the fix, and it still reproduces #1756.

**Comment by timbze** [comment]
Google chrome can easily uninstall updates in Android 8.x - that's what uninstall from Play Store does since it's a system app.

Maybe there should be an advanced option in settings to "Disable hardware acceleration", which would be enabled for 8 or 8.1, but then there would be the option for those who want to uninstall chrome updates and then enable hardware acceleration.

**Comment by tuomas2** [comment]
I have not tried to really "use" And Bible on 8 device, BUT I have tried doing bookmarks etc with lenovo tablet that I have borrowed from a friend and it is not *that* slow.

**Comment by tuomas2** [comment]
User that downgraded because of slowness on Android 8 https://support.andbible.org/scp/tickets.php?id=936

**Comment by tuomas2** [comment]
[Google Play reviewer reports](https://play.google.com/console/u/0/developers/6727655833913220393/app/4974350217168681006/user-feedback/review-details?reviewId=gp%3AAOqpTOGJFRorRUkz1G_FHMFQMAGLlcbx2KFQal6BXsQCytYeef-3uil559a9ZK-bR-4iDYj20qQ7rzLtlpM&corpus=PUBLIC_REVIEWS)

**Comment by tuomas2** [comment]
User is telling he does not have issues with 605 https://support.andbible.org/scp/tickets.php?id=948

**Comment by timbze** [comment]
> User is telling he does not have issues with 605 https://support.andbible.org/scp/tickets.php?id=948

He may have older webview. Looks like we don't have full info about his device

**Comment by tuomas2** [comment]
From logcat: 10-17 11:15:59.040 31992 31992 I WebViewFactory: Loading com.android.webview version 91.0.4472.158 (code 447215803)

**Comment by timbze** [comment]
> From logcat: 10-17 11:15:59.040 31992 31992 I WebViewFactory: Loading com.android.webview version 91.0.4472.158 (code 447215803)

The lowest webview version that I find we got an error report about was 93.

**Comment by tuomas2** [comment]
Can someone check if issue #1756 still reproduces with the latest Chrome installed on Anroid 8 and version https://github.com/AndBible/and-bible/releases/tag/beta-607.1

**Comment by timbze** [comment]
Yes just checked that the issue still exists in Chrome Canary 102.0.4990.0

**Comment by tuomas2** [comment]
Can't reproduce on Honor 8 lite (Android 8 phone, which I just saved from being trashed at my working place). Latest Chrome installed. Tested with latest AB with Android 8 workaround disabled.

**Comment by tuomas2** [comment]
https://support.andbible.org/scp/tickets.php?id=1805

**Comment by meichthys** [comment]
It seems like we could probably let this one go at this point unless it reproduces on android 9/10+ ?
Android 8 was end-of-life 5 years ago.
